### PR TITLE
Fixes for clipping

### PR DESCRIPTION
**Pull request by sloriot**
- better handling of border non-manifold edges during clipping.

### CODE REVIEW

File: Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_clip.cpp [hunk 1]
```diff
@@ -283,6 +284,46 @@ void test()
   assert(is_valid_polygon_mesh(tm1));
   CGAL::clear(tm1);
   CGAL::clear(tm2);
+
+  // non-manifold border vertices
+  std::stringstream ss;
+  ss << "OFF\n 5 2 0\n 0 0 0\n2 0 0\n4 0 0\n4 1 0\n0 1 0\n3 0 1 4\n3 1 2 3\n";
```
Comment (lrineau) [inline Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_clip.cpp]: Note that C++11 has raw string literals, that allows multi-lines strings.
https://en.cppreference.com/w/cpp/language/string_literal#Example

Comment (sloriot) [reply Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_clip.cpp]: Milestone is 4.14.1

Comment (sloriot) [reply Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_clip.cpp]: But thanks for the ref :)

**Comment by maxGimeno** [comment]
[Execution errors](https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-77.shtml#Polygon_mesh_processing)

**Comment by sloriot** [comment]
I forgot to add the new files.

**Comment by sloriot** [comment]
Wrong branch, I restored the flags

**Comment by maxGimeno** [comment]
[warnings](https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-82.shtml#Polygon_mesh_processing)

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-87.shtml

**Pull request merged**